app/src/main/res/layout/offer_header.xml

```diff
@@ -7,10 +7,11 @@
     android:paddingBottom="@dimen/base_margin_quadruple"
     tools:context=".feature.offer.ui.OfferActivity">
 
-    <com.hedvig.app.ui.view.HedvigCardView
+    <androidx.cardview.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/base_margin_double">
+        android:layout_margin="@dimen/base_margin_double"
+        app:cardCornerRadius="@dimen/hedvig_card_view_radius">
```

Comment:
```suggestion
        app:cardCornerRadius="@dimen/hedvig_card_view_radius">
```

Something's strange if we're not getting this from the theme, it should be consistent throughout the app 😄

Reply:
Changed to MaterialCardView and the radius was correct :) Removing this